app/models/product.rb

```diff
@@ -0,0 +1,3 @@
+class Product < ApplicationRecord
+  scope :recent, ->{order(created_at: :desc)}
```

Comment:
```suggestion
  scope :recent, -> {order(created_at: :desc)}
```
Trước block nên có 1 space